spec/e2e/migrations_spec.rb

```diff
@@ -0,0 +1,113 @@
+describe Neo4j::Migrations::Runner do
+  before(:all) { ENV['silenced'] = 'true' }
+  after(:all)  { ENV['silenced'] = nil }
```

Comment:
It's not a big deal for this I think, but we might think about having a `let_env_variable` helper which would work something like `let_env_variable('silenced', true)` and it would store and restore the variable back to it's original state after the spec is done (similar to `let_config`)